frame/frame_test_case.py

```diff
@@ -88,6 +88,7 @@ def setUp(self) -> None:
             self.frame.kill()
             raise
         self.start_time = time.time()
+        os.environ['QT_QPA_PLATFORM'] = 'wayland'
```

Comment:
You may want to pass `env=os.environ | {'QT_QPA_PLATFORM': 'wayland'}` to `subprocess.Popen` to avoid polluting the current environment?